### ISSUE TITLE
Only see 3 resolutions - Did I do something wrong?  on Windows 11

### ISSUE THREAD

**Issue by planet** [itsmikethetech/Virtual-Display-Driver#3]
![image](https://github.com/itsmikethetech/Virtual-Display-Driver/assets/1265868/0f5b713f-79df-4aad-950d-60d5c816f057)

**Comment by planet** [comment]
Nevermind this issue occured when I copied the latest option.txt outside of the package. If I use the included options.txt, it appears to be ok.

**Comment by chauser10** [comment]
@planet Having the same issue. What do you mean by "latest option.txt outside of the package"?

**Comment by GreatWizard** [comment]
If you use the options.txt from the repository source or the latest preview version: you have this issue.
We should reopened the ticket.

**Comment by planet** [comment]
Sure, i'll re-open.

**Comment by itsmikethetech** [comment]
Thank you for the report. The preview version is confirmed to have some issues with display resolutions related to the options.txt file. Reverted to the main version and working on an updated file with more consistent support. This is related to the way I had tried to implement multi-refresh rate support without having to add hundreds of lines to the file. Will update by next week. Thanks for the reports!
